pkg/apis/crds/karpenter.sh_nodepools.yaml

```diff
@@ -78,7 +81,15 @@ spec:
                       node upgrade, memory leak protection, and disruption testing.
                     pattern: ^(([0-9]+(s|m|h))+)|(Never)$
                     type: string
+                    x-kubernetes-validations:
+                    - message: expireAfter cannot be negative
+                      rule: '!self.startsWith(''-'')'
                 type: object
+                x-kubernetes-validations:
+                - rule: 'has(self.consolidateAfter) ? self.consolidationPolicy!=''WhenUnderutilized''
```

Comment:
Just curious. Can you do spaces in between `!=` and `==` or do you have to have no spaces in there